File: app/src/main/java/com/smartbadge/app/domain/model/VisitSummary.kt
package com.smartbadge.app.domain.model

data class VisitSummary(
    val topics: List<String> = emptyList(),
    val conclusions: List<String> = emptyList(),
    val todos: List<TodoItem> = emptyList(),
    val nextSteps: String = ""
)

data class TodoItem(
    val task: String,
    val owner: String = "",
    val deadline: String = ""
)

File: ios/VoiceNote/VoiceNote/Domain/Model/VisitSummary.swift
import Foundation

/// AI 生成的总结
struct RecordSummary: Codable, Equatable {
    var topics: [String]
    var conclusions: [String]
    var todos: [TodoItem]
    var nextSteps: [String]
}

/// 待办事项
/// 对齐 Android: TodoItem
struct TodoItem: Codable, Equatable, Identifiable {
    let id: UUID
    var task: String
    var owner: String
    var deadline: String

    init(id: UUID = UUID(), task: String = "", owner: String = "", deadline: String = "") {
        self.id = id
        self.task = task
        self.owner = owner
        self.deadline = deadline
    }
}
